File: program/program/check-consonant/check_consonant.go
package main

import "fmt"

func main() {
	vowels := []string{"a", "e", "i", "o", "u"}
	var char string
	fmt.Scan(&char)
	flag := true
	for _, v := range vowels {
		if v == char {
			flag = false
			break
		}
	}
	if flag {
		fmt.Println("Consonant")
	} else {
		fmt.Println("Vowel")
	}
}

File: program/program/check-consonant/CheckConsonant.cs
using System;
public class LFC
{
  public static void Main(string[] args)
  {
    char ch='b';
    if((ch >= 'A' && ch <= 'Z')||ch >= 'a' && ch <= 'z')
    {
      if (ch == 'a' || ch == 'A' || ch == 'e' || ch == 'E' || ch == 'i'
      || ch == 'I' || ch == 'o' || ch == 'O' || ch == 'u' || ch == 'U')
      {
        Console.WriteLine(ch + " is a vowel");
      }
      else
      {
        Console.WriteLine(ch + " is a consonant");
      }
    }
    else
    {
      Console.WriteLine(ch + " is not an alphabet");
    }
  }
}
